File: app/src/main/java/br/com/popularmovies/services/movieService/response/Movies.java
package br.com.popularmovies.services.movieService.response;


import com.squareup.moshi.Json;

import java.util.List;

public class Movies {

    @Json(name = "results")
    private final List<Movie> movies;

    public Movies(List<Movie> movies) {
        this.movies = movies;
    }

    public List<Movie> getMovies() {
        return movies;
    }
}

File: app/src/main/java/br/com/popularmovies/services/movieService/response/Movies.kt
package br.com.popularmovies.services.movieService.response

import android.os.Parcelable
import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import kotlinx.android.parcel.Parcelize

@Parcelize
@JsonClass(generateAdapter = true)
data class Movies(@Json(name = "results") val movieDtos: List<MovieDto>) : Parcelable